lib/utils/addEntries.js

```diff
@@ -70,6 +69,14 @@ function addEntries(config, options, server) {
         }
       }
     });
+    if (Array.isArray(config.plugins)) {
+      config.plugins.map((item) => {
+        if (item.definitions) {
+          item.definitions.SOCKJS_PORT = options.port || 8080;
+        }
+        return item;
+      });
+    }
```

Comment:
Remove this it is invalid

Reply:
i also think it is invalid
but when i commit eslint promot arrow function must have a return value -。-

Reply:
No it is invalid code all :smile:

Reply:
already remove